src/components/jacobian_component/jacobian.sh

```diff
@@ -44,12 +55,21 @@ setup_jacobian() {
     sed -i -e "s:{RunName}:${RunName}:g" \
            -e "s:{InversionPath}:${InversionPath}:g" jacobian_runs/run_prior_simulation.sh
 
+    # Modify HEMCO_Config.rc to turn off individual emission inventories
+    # and use total emissions saved out from prior emissions simulation
+    # instead
+    # Do this in template run directory to avoid having to repeat for each
+    # Jacobian run directory
+    printf "\nTurning on use of total prior emissions in HEMCO_Config.rc.\n"
+    sed -i -e "s|UseTotalPriorEmis      :       false|UseTotalPriorEmis      :       true|g" \
+           -e "s|AnalyticalInversion    :       false|AnalyticalInversion    :       true|g" \
```

Comment:
Was use analytical inversion false before?

Reply:
`AnalyticalInversion` is false by default when setting up a GEOS-Chem run directory for CH4 simulations. It remained false in the IMI prior to these updates. This is because we were previously applying the emissions perturbations in the GEOS-Chem source code (GeosCore/global_ch4_mod.F90). That code has since been removed and we now apply emissions perturbations directly in HEMCO_Config.rc by setting `AnalyticalInversion: true`.